components/contribution-flow/ContributionSummary.js

```diff
@@ -78,11 +78,18 @@ const ContributionSummary = ({ collective, stepDetails, stepSummary, stepPayment
               />
             </Label>
             <Amount>
-              <FormattedMoneyAmount
-                amount={stepDetails.amount}
-                currency={currency}
-                amountStyles={{ color: 'black.700', fontWeight: 400 }}
-              />
+              {!isCrypto && (
+                <FormattedMoneyAmount
+                  amount={stepDetails.amount}
+                  currency={currency}
+                  amountStyles={{ color: 'black.700', fontWeight: 400 }}
+                />
+              )}
+              {isCrypto && (
+                <Span
```

Comment:
Why not handle this in FormattedMoneyAmount?

Reply:
Same reason as https://github.com/opencollective/opencollective-frontend/pull/6689#discussion_r682151172

Reply:
Please change.

Reply:
Changed. 👍🏽